### ISSUE TITLE
Define install/release procedure for template-only Connectors

### ISSUE THREAD

**Issue by chillleader** [camunda/connectors#128]
* Determine how template-only connectors differ from 'full' connectors in terms of releasing
* Make adjustments to CI workflows
* Document the procedure

see https://github.com/camunda/connectors-bundle/pull/125 for context

**Comment by sbuettner** [comment]
The current procedure to manually update element templates in the modeler could also be improved by having an artifact that could be distributed and consumed more easily.

**Comment by sbuettner** [comment]
We should consider the customer feedback coming from https://camunda.slack.com/archives/C02JLRNQQ05/p1682433886586029

>  They were a little unclear why in the self-managed Web Modeler the ootb connectors are not installed by default. Additionally, I personally found the installation instructions a bit hard to follow. I think it misses the part where one publishes the connectors to make them available in a project? : https://docs.camunda.io/docs/self-managed/connectors-deployment/install-and-start/#connector-templates

**Comment by bastiankoerber** [comment]
Marketplace will solve this issue
